interface/patient_file/summary/demographics.php

```diff
@@ -1017,7 +1017,7 @@ function setMyPatient() {
                                                             <?php echo text($row['subscriber_ss']); ?><br />
                                                             <?php echo xlt('D.O.B.'); ?>:
                                                             <?php
-                                                            if ($row['subscriber_DOB'] != "0000-00-00 00:00:00") {
+                                                            if ($row['subscriber_DOB'] != null) {
```

Comment:
When something is null in the database, it will just return an empty variable, so you could now actually just remove this if now and always run the `echo text($row['subscriber_DOB']);`